Assets/Scripts/SS3D/Systems/Tile/Enums/TileLayer.cs

```diff
@@ -13,11 +13,14 @@ public enum TileLayer
         Turf,
         Wire,
         Disposal,
-        Pipes,
+        PipeSurface,
```

Comment:
the issue with unity and enums is that when you modify the order in the enum, Unity gets confused with the enums you registerer in editor. I did not put them together because I was afraid to break something. It's probably not much change but that means retesting a lot of stuff, can we do a separate PR for that ? If you insist I will change it though.